pods.go

```diff
@@ -159,26 +173,15 @@ func name2ord(name string) int {
 }
 
 // genpod generates the pod specification
-func genpod(namespace, name, image string) *corev1.Pod {
-	var userID int64 = 65534
-
-	return &corev1.Pod{
-		Metadata: &metav1.ObjectMeta{
-			Name:      k8s.String(name),
-			Namespace: k8s.String(namespace),
-			Labels:    map[string]string{"generator": "kboom"},
-		},
-		Spec: &corev1.PodSpec{
-			Containers: []*corev1.Container{
-				&corev1.Container{
-					Name:    k8s.String("main"),
-					Image:   k8s.String(image),
-					Command: []string{"/bin/sh", "-ec", "sleep 3600"},
-					SecurityContext: &corev1.SecurityContext{
-						RunAsUser: &userID,
-					},
-				},
-			},
-		},
+func genpod(namespace, name, image string, podTemplate corev1.Pod) *corev1.Pod {
+
+	podTemplate.Metadata = &metav1.ObjectMeta{
+		Name:      k8s.String(name),
+		Namespace: k8s.String(namespace),
+		Labels:    map[string]string{"generator": "kboom"},
 	}
+	// TODO: currently only one container is supported
+	podTemplate.Spec.Containers[0].Image = k8s.String(image)
```

Comment:
Do we still need this feature if we can provide a custom Pod Template ? From my pov we could drop this functionality (which is currently not documented anyway :))